client/my-sites/site-settings/disconnect-site/down-flow.jsx

```diff
@@ -76,9 +76,9 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
 					<FormattedHeader
 						isSecondary
 						align="left"
-						headerText={ translate( 'Troubleshoot Jetpack.' ) }
+						headerText={ translate( 'Troubleshoot Jetpack' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.